### DISCUSSION TITLE
server install

### DISCUSSION THREAD

**Discussion by punkyard** [exoplatform/exo-community#10] [Q&A]
hi
is there a set-up guide for server installation?
I'd like to test EXO on an ubuntu VPS, 4cores 8gigs
thanks!

**Comment by ahamdi** [comment]
Hi @punkyard 
You can follow this documentation https://exoplatform.github.io/guide/getting-started/start-community.html
It will guide you how to start the package with all its components
Regards

**Reply by punkyard** [reply]
thanks @ahamdi
